source/components/text-input/index.html

```diff
@@ -5,157 +5,100 @@
-        <input type="text" id="txtDisabled" placeholder="Text" disabled>
-      </div>
-    </form>
-  </div>
-  {% code 'html' %}
-    <form>
-      <div class="hxFormGroup">
-        <label for="txt">Default</label>
-        <input type="text" id="txt" placeholder="Text">
-      </div>
-      <div class="hxFormGroup">
-        <label for="txtInvalid">Invalid</label>
-        <input type="text" class="-hxInvalid" id="txtInvalid" placeholder="Text">
-      </div>
-      <div class="hxFormGroup">
-        <label for="txtDisabled">Disabled</label>
-        <input type="text" id="txtDisabled" placeholder="Text" disabled>
-      </div>
-    </form>
-  {% endcode %}
+  <p>
+    Use the <code>.hxTextCtrl</code> CSS class to apply styles to text-like form controls.
+  </p>
 </section>
 
 <section>
-  <h2 class="hxSectionTitle" id="email">Email</h2>
+  <h2 class="hxSectionTitle" id="single-line-text">Single-Line Text</h2>
   <div class="demo">
-    <form>
-      <div class="hxFormGroup">
-        <label for="txtEmail">Default</label>
-        <input type="email" id="txtEmail" placeholder="Email">
-      </div>
-      <div class="hxFormGroup">
-        <label for="txtInvalidEmail">Invalid</label>
-        <input type="email" class="-hxInvalid" id="txtInvalidEmail" placeholder="Email">
-      </div>
-      <div class="hxFormGroup">
-        <label for="txtDisabledEmail">Disabled</label>
-        <input type="email" id="txtDisabledEmail" placeholder="Email" disabled>
-      </div>
-    </form>
-  </div>
-  {% code 'html' %}
-    <form>
-      <div class="hxFormGroup">
-        <label for="txtEmail">Default</label>
-        <input type="email" id="txtEmail" placeholder="Email">
-      </div>
-      <div class="hxFormGroup">
-        <label for="txtInvalidEmail">Invalid</label>
-        <input type="email" class="-hxInvalid" id="txtInvalidEmail" placeholder="Email">
-      </div>
-      <div class="hxFormGroup">
-        <label for="txtDisabledEmail">Disabled</label>
-        <input type="email" id="txtDisabledEmail" placeholder="Email" disabled>
-      </div>
-    </form>
-  {% endcode %}
-</section>
+    <input
+      type="text"
+      class="hxTextCtrl"
+      placeholder="Default Text Input" />
+    <br />
 
-<section>
-  <h2 class="hxSectionTitle" id="password">Password</h2>
-  <div class="demo">
-    <form>
-      <div class="hxFormGroup">
-        <label for="txtPassword">Password</label>
-        <input type="password" id="txtPassword" placeholder="Password">
-      </div>
-      <div class="hxFormGroup">
-        <label for="txtInvalidPassword">Invalid</label>
-        <input type="password" class="-hxInvalid" id="txtInvalidPassword" placeholder="Password">
-      </div>
-      <div class="hxFormGroup">
-        <label for="txtDisabledPassword">Disabled</label>
-        <input type="password" id="txtDisabledPassword" placeholder="Password" disabled>
-      </div>
-    </form>
+    <input
+      type="text"
+      class="hxTextCtrl -hxInvalid"
```

Comment:
updated `-hxInvalid` to use attribute `invalid` similar to usage in checkbox

Reply:
done